### ISSUE TITLE
Show number of words behind the loglength.

### ISSUE THREAD

**Issue by epnl** [2Abendsegler/GClh#1273]
As loglength in words is a badge issue, it would be nice to have the log length in number of words along the number of characters.

**Comment by capoaira** [comment]
I can implement this.

**Comment by 2Abendsegler** [comment]
Show number of words behind the loglength on both log forms.

Something like this: 
![grafik](https://user-images.githubusercontent.com/22332216/74473338-bc0fa880-4ea3-11ea-8717-58b57075c4c0.png)
Maybe put the number of words at the back, or only after the number of letters, I don't know yet.
Maybe set title, if you go with the mouse about the fields, which figures mean what.

@epnl 
Would that be okay?

**Comment by 2Abendsegler** [comment]
@capoaira 
And what is with the issues 1113 and 1117? 🤨

**Comment by capoaira** [comment]
> Maybe put the number of words at the back, or only after the number of letters, I don't know yet.
> Maybe set title, if you go with the mouse about the fields, which figures mean what.

That's how I wanted to do it. Possibly write "words" after the number so that it becomes clear with the first view. I also set the title.


> @capoaira
> And what is with the issues 1113 and 1117? 🤨

I had no new Ideas for this Issues in the last time.

**Comment by 2Abendsegler** [comment]
@capoaira 
> Possibly write "words" after the number so that it becomes clear with the first view. I also set the title.

Maybe we need no title if we write "words".
Please wait with the development until @epnl has give us a go.

**Comment by capoaira** [comment]
@epnl which solution do you like? Please give a feedback so I can start developing.

**Comment by epnl** [comment]
Oh thanks for you offering me a say in this!

I suppose
Loglength: 13/4000 (2 words) 
would be fine.  :)

Thanks!

**Comment by capoaira** [comment]
Should I set the `font-weight` to `normal` on the old logging form? 
With `font-weight: bold` 
![2020-02-24](https://user-images.githubusercontent.com/50927164/75166494-39e57680-5724-11ea-9ee1-b1b1efa0ffaf.png)

But I think it looks better with `font-weight: normal` because "words" does not jump to the next line and it is then like the new logging form.
![2020-02-24 (1)](https://user-images.githubusercontent.com/50927164/75166521-41a51b00-5724-11ea-9d7c-aa30b673cedb.png)

What do you think @2Abendsegler?

**Comment by 2Abendsegler** [comment]
You are right, normal should be netter.

**Comment by 2Abendsegler** [comment]
Sehr schön, paßt!

---
```
	<li>
		<strong>New:</strong> Show number of words behind the loglength. [<a href="https://github.com/2Abendsegler/GClh/issues/1273" title="Issue 1273">1273</a> / <a href="https://www.geocaching.com/profile/?u=capoaira" title="Thanks to capoaira">capoaira</a>]<br>
		Old log form:<br>
		<img src="../images/0.10.5/Screen10.jpg" alt="Screen10.jpg"><br>
		New log form:<br>
		<img src="../images/0.10.5/Screen11.jpg" alt="Screen11.jpg"></a><br><br>
	</li>
```

![Screen10](https://user-images.githubusercontent.com/22332216/75188592-aecca680-574c-11ea-8bc6-5a14bbe43dfc.jpg)

![Screen11](https://user-images.githubusercontent.com/22332216/75188599-b1c79700-574c-11ea-8562-6e666aac622b.jpg)

**Comment by epnl** [comment]
Wonderful!
Sieht sehr schön aus!
